server/app/views/admin/programs/ProgramApplicationTableView.java

```diff
@@ -438,7 +438,7 @@ private DivTag renderApplicationTable(
                           alertTag,
                           applicationTable)));
     }
-    return div(applicationTable);
+    return div(applicationTable).withData("testid", "application-table");
```

Comment:
Actually, it is getting used in the next PR for the browser tests. Since that PR is too big, I fragmented it to server changes in one PR and browser tests in another.

Reply:
I will be using it in the browser tests PR coming right ahead - https://github.com/civiform/civiform/pull/9384